src/main/java/com/fontys/api/queries/TeamQuery.java

```diff
@@ -18,7 +18,7 @@ public TeamQuery(TeamService teamService) {
         this.teamService = teamService;
     }
 
-    public List<Team> teams() {
+    public List<Team> teams(int count) {
```

Comment:
can this be made a nullable parameter or with a default value?

Reply:
On it.

Reply:
Thought Java does not support nullable ints. After some research by Lloyd, turns out you can use the integer **class** and use the @Nullable annotation.
Now made nullable.